.github/actions/publish_anaconda/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Anaconda
+description: Publish a noarch python package to anaconda.org
+inputs:
+  anaconda_token:
+    required: true
+    description: The anaconda token to use for upload
+
+  dist_path:
+    required: true
+    description: A path to the python dist tarball to be uploaded
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v2.2.0
+
+    - name: Install Anaconda client
+      shell: bash -el {0}
+      run: conda install -y anaconda-client
```

Comment:
Looks like you can get this from pip:
https://github.com/Anaconda-Platform/anaconda-client#with-pip

So the whole publish action could be replaced by:
`pipx run --spec anaconda-client anaconda --token ${{ secrents.ANACONDA_TOKEN }} upload conda_build/noarch/*.tar.bz2

Reply:
Ooh, that's really nice, I really need to start making better use of pipx